google-ads/src/test/resources/testdata/avail_service_clients.txt

```diff
@@ -0,0 +1,107 @@
+createAccountBudgetProposalServiceClient: AccountBudgetProposalServiceClient
```

Comment:
Would it be reasonably simple to extend this to cover method signatures as well? I'm thinking that we can use this to ensure the flattenings we're expecting are being generated correctly.